nva-datamodel-testutils/src/main/java/no/unit/nva/model/testing/PublicationGenerator.java

```diff
@@ -82,10 +91,33 @@ public static Publication randomPublicationNonDegree() {
         return randomPublication(randomElement(nonDegrees));
     }
 
+    public static Publication randomPublicationNonDegreeThesis() {
```

Comment:
do we need this?

Reply:
maybe add javadoc comment that explains difference

Reply:
refactored